Makefile

```
@@ -1,24 +1,28 @@
 .PHONY: dev ui schema annotations check package
 
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)